Exiled.API/Extensions/MirrorExtensions.cs

```diff
@@ -141,6 +141,18 @@ public static class MirrorExtensions
         /// </summary>
         public static MethodInfo SendSpawnMessageMethodInfo => sendSpawnMessageMethodInfoValue ??= typeof(NetworkServer).GetMethod("SendSpawnMessage", BindingFlags.NonPublic | BindingFlags.Static);
 
+        /// <summary>
+        /// Add an Operation to execute on a <see cref="SyncList{T}"/>.
+        /// </summary>
+        /// <param name="synclist">Only this player can see info.</param>
+        /// <param name="op">Operation to execute.</param>
+        /// <param name="itemIndex">Item index to affect.</param>
+        /// <param name="oldItem">previous item.</param>
+        /// <param name="newItem">new item.</param>
+        /// <param name="checkAccess">Verify if access is authorised.</param>
+        /// <typeparam name="T">A type of <see cref="SyncList{T}"/>.</typeparam>
+        public static void AddOperation<T>(this SyncList<T> synclist, SyncList<T>.Operation op, int itemIndex, T oldItem, T newItem, bool checkAccess) => typeof(SyncList<T>).GetMethod("AddOperation", BindingFlags.Instance | BindingFlags.NonPublic)?.Invoke(synclist, new object[] { op, itemIndex, oldItem, newItem, checkAccess });
```

Comment:
save the method, do not .GetMethod every time

Reply:
Do pr

Reply:
not possible afaik, it would give a TargetInvocationException because before we were saving SyncList<> method but we need SyncList<T> method

Reply:
two of the properties in the method need the type